src/BoundaryConditions/polar_boundary_condition.jl

```diff
@@ -0,0 +1,76 @@
+const PolarBoundaryCondition{V} = BoundaryCondition{<:Value, <:PolarValue}
+
+condition_operand(data, grid, Loc, condition, mask) = data
+
+struct PolarValue{C}
+    c :: C
+end
+
+PolarBoundaryCondition(field) = 
+    ValueBoundaryCondition(PolarValue(field))
+
+@inline getbc(pv::PolarValue, args...) = getbc(pv.c, args...)
+
+# TODO: vectors should have a different treatment since vector components should account for the frame of reference
+# North - South flux boundary conditions are not valid on a Latitude-Longitude grid if the last / first rows represent the poles
+function regularize_north_boundary_condition(bc::DefaultBoundaryCondition, grid::LatitudeLongitudeGrid, loc, dim, args...) where C
+    φmax = φnode(grid.Ny+1, grid, Face()) 
+    
+    if φmax == 90 
+        _, LY, LZ = loc
+        field = Field{Nothing, LY, LZ}(grid; indices = (:, grid.Ny, :))
+        return PolarBoundaryCondition(field)
+    else
+        return regularize_boundary_condition(bc, grid, args...)
+    end
+end
+
+# North - South flux boundary conditions are not valid on a Latitude-Longitude grid if the last / first rows represent the poles
+function regularize_south_boundary_condition(bc::DefaultBoundaryCondition, grid::LatitudeLongitudeGrid, loc, dim, args...) 
+    φmin = φnode(1, grid, Face()) 
+
+    if φmin == - 90 
+        _, LY, LZ = loc
+        field = Field{Nothing, LY, LZ}(grid; indices = (:, 1, :))
```

Comment:
This does seem right

Reply:
But should `LY` actually be flipped? Eg this is Face for a Center location. (It may not matter)

Reply:
Probably you are right. I have found out unfortunately that reductions on windowed fields do not work. I ll try to make it work so here we can do
```julia
mean!(windowed_field, full_field`
```